Resources/Textures/Objects/Misc/Lights/lamp.rsi/meta.json

```
@@ -1,33 +1,35 @@
-{
-  "version": 1,
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/fb020482a6737372d5b8f2b603d58737b22b0270",
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-        "name": "lamp"
-    },
-    {
-        "name": "lamp-on"
-    },
-    {
-      "name": "off-inhand-left",
-      "directions": 4
-    },
-    {
-      "name": "off-inhand-right",
-      "directions": 4
-    },
-    {
-      "name": "on-inhand-left",
-      "directions": 4
-    },
-    {
-      "name": "on-inhand-right",
-      "directions": 4
-    }
-  ]
-}
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/fb020482a6737372d5b8f2b603d58737b22b0270",

```

Добавь и в другие файлы
```suggestion
  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/fb020482a6737372d5b8f2b603d58737b22b0270, directional sprites by SonicDC#9046",

```